tests/integration/test_async_http_client.py

```diff
@@ -7,16 +7,15 @@
 from solana.rpc.api import DataSliceOpt
 from solana.rpc.async_api import AsyncClient
 from solana.rpc.commitment import Finalized
-from solana.rpc.core import RPCException
-from solana.rpc.types import RPCError
+from solana.rpc.core import RPCException, TransactionExpiredBlockheightExceededError, TransactionUncompiledError
+from solana.rpc.types import RPCError, TxOpts
 from solana.transaction import Transaction
 from spl.token.constants import WRAPPED_SOL_MINT
 
 from .utils import AIRDROP_AMOUNT, assert_valid_response
 
 
 @pytest.mark.integration
-@pytest.mark.asyncio
```

Comment:
Why are we removing these decorators?

Reply:
[pytest-asyncio's latest version](https://github.com/pytest-dev/pytest-asyncio#modes) does not require it anymore if async_mode=auto in pytest.ini